hikari/presences.py

```diff
@@ -118,11 +121,16 @@ class ActivityParty:
     """Maximum size of this party, if applicable."""
 
 
+_DYNAMIC_URLS = {"mp": urls.MEDIA_PROXY_URL + "/{}"}
```

Comment:
I am pretty confident discord will never add any more dynamic URLs and, if they do, it will be behind their proxy, so this might be a bit redundant

Reply:
How can you be confident in something that's already not the case?

Reply:
Because they seem to be doing everything through one of their services. They are all already proxied here, for like the 4 services they provide atm + the ones they announced, which is basically anyone else that feels like using this with the SDK.

Reply:
> Because they seem to be doing everything through one of their services.

That's just not true? Right now you can find dynamic links on presences in production for "twitch", "mp",  "youtube" and "spotify" (and i highly doubt twitch, youtube or spotify are coming from the SDK); this PR just avoids including 3 of these right now as these are undocumented

Reply:
I got confused with nights last commit to the PR before merging it (https://github.com/discord/discord-api-docs/pull/4317/commits/537a493572d2086a8b7bd0b3808f0c18b935e495) and thought he had merged all of the documented ones into the `mp` one, looks like they just removed all of them but `mp`. Sorry for the confusion